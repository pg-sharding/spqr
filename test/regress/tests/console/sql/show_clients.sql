SHOW clients;
SHOW clients WHERE server_id = spqr_shard_1:6432;
SHOW clients WHERE server_id = nonexistent:6432 OR dbname = regress;