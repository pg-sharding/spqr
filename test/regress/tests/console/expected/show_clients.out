
		SPQR router admin console
	Here you can configure your routing rules
------------------------------------------------
	You can find documentation here 
https://github.com/pg-sharding/spqr/tree/master/docs

SHOW clients;
 client_id | user | dbname | server_id | router_address | router_time_0.75 | shard_time_0.75 
-----------+------+--------+-----------+----------------+------------------+-----------------
(0 rows)

SHOW clients WHERE server_id = spqr_shard_1:6432;
 client_id | user | dbname | server_id | router_address | router_time_0.75 | shard_time_0.75 
-----------+------+--------+-----------+----------------+------------------+-----------------
(0 rows)

SHOW clients WHERE server_id = nonexistent:6432 OR dbname = regress;
 client_id | user | dbname | server_id | router_address | router_time_0.75 | shard_time_0.75 
-----------+------+--------+-----------+----------------+------------------+-----------------
(0 rows)

